packages/smooth_app/lib/helpers/permission_helper.dart

```diff
@@ -23,6 +25,13 @@ class PermissionListener extends ValueNotifier<DevicePermission> {
   Future<void> askPermission(
     Future<bool?> Function() onRationaleNotAvailable,
   ) async {
+    // Prevent multiples calls to this method
+    if (_status == _DevicePermissionStatus.asked) {
```

Comment:
Just curious: could that rather be `if (_status != _DevicePermissionStatus.initial) {`?

Reply:
(It think, you've asked twice the same question. Closing this one)